<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Bankim" failures="0" tests="7" name="com.ny.getemployeeinfo.Put_Method" time="0.010" errors="1" timestamp="10 Mar 2023 15:34:59 GMT" skipped="6">
  <testcase classname="com.ny.getemployeeinfo.Put_Method" name="checkResponseBody" time="0.000">
    <skipped/>
  </testcase> <!-- checkResponseBody -->
  <testcase classname="com.ny.getemployeeinfo.Put_Method" name="checkStatusLine" time="0.000">
    <skipped/>
  </testcase> <!-- checkStatusLine -->
  <testcase classname="com.ny.getemployeeinfo.Put_Method" name="updateEmployee" time="0.010">
    <error message="Cannot invoke &quot;org.apache.log4j.Logger.info(Object)&quot; because &quot;com.ny.getemployeeinfo.Put_Method.log&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.log4j.Logger.info(Object)" because "com.ny.getemployeeinfo.Put_Method.log" is null
at com.ny.getemployeeinfo.Put_Method.updateEmployee(Put_Method.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- updateEmployee -->
  <testcase classname="com.ny.getemployeeinfo.Put_Method" name="checkContentType" time="0.000">
    <skipped/>
  </testcase> <!-- checkContentType -->
  <testcase classname="com.ny.getemployeeinfo.Put_Method" name="checkStatusCode" time="0.000">
    <skipped/>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="com.ny.getemployeeinfo.Put_Method" name="checkTitle" time="0.000">
    <skipped/>
  </testcase> <!-- checkTitle -->
  <testcase classname="com.ny.getemployeeinfo.Put_Method" name="checkContentEncoding" time="0.000">
    <skipped/>
  </testcase> <!-- checkContentEncoding -->
</testsuite> <!-- com.ny.getemployeeinfo.Put_Method -->
